packages/mui-codemod/src/deprecations/all/deprecations-all.js

```diff
@@ -1,11 +1,15 @@
-import transformAccordionProps from '../accordion-props/accordion-props';
+import transformAccordionProps from '../accordion-props';
+import transformAvatarProps from '../avatar-props';
+import transformDividerProps from '../divider-props';
 
 /**
  * @param {import('jscodeshift').FileInfo} file
  * @param {import('jscodeshift').API} api
  */
 export default function deprecationsAll(file, api, options) {
   file.source = transformAccordionProps(file, api, options);
+  file.source = transformAvatarProps(file, api, options);
+  file.source = transformDividerProps(file, api, options);
```

Comment:
@DiegoAndai For completeness, I added divider transform here. Let me know if this is okay to you, otherwise I will open a separate PR.

Reply:
That's perfect, we missed it 😅